cm_sdk/models/alert_config/alert_config.py

```diff
@@ -0,0 +1,92 @@
+from cm_sdk.models import CloudManagerBase, CloudManagerEnum
+from cm_sdk.models.alert_config.matcher import Matcher
+from cm_sdk.models.alert_config.notifications import Notification
+from cm_sdk.models.alert_config.metric_threshold import MetricThreshold
+from cm_sdk.models.common.link import Link
+
+name_map = {"type_name":"TypeName"}
+
+class TypeName(CloudManagerEnum):
```

Comment:
You have a class name `TypeName` both here and in the [alert.py class](https://github.com/edx-ops/edx-cloudmanager-sdk/pull/6/files#diff-140363828b472423ec487cf71f1516e9R14).  That one looks like it's a nested class but they have the same name and the values don't differ by very much.  Is this on purpose?